app/internal/wm/os_android.go

```diff
@@ -728,6 +753,20 @@ func setCursor(env *C.JNIEnv, view C.jobject, name pointer.CursorName) {
 	callVoidMethod(env, view, gioView.setCursor, jvalue(curID))
 }
 
+func setStatusColor(env *C.JNIEnv, view C.jobject, color color.NRGBA) {
+	callVoidMethod(env, view, gioView.setStatusColor,
+		jvalue(uint32(color.A)<<24|uint32(color.R)<<16|uint32(color.G)<<8|uint32(color.B)),
+		jvalue(int64((0.2126*float64(color.R))+(0.7152*float64(color.G))+(0.0722*float64(color.B)))),
```

Comment:
The "luminance" is used to switch the button/font color. Maybe add some comments about it? Or let it customizable, so you can choose between White or Black icons, on the Status-Bar?.

Reply:
As long as you're ok with the luminance-based selection between dark and light, let's keep the API simple.